app/file.go

```diff
@@ -553,18 +553,16 @@ func getImageOrientation(input io.Reader) (int, error) {
 }
 
 func generateThumbnailImage(img image.Image, thumbnailPath string, width int, height int) {
-	thumbWidth := float64(utils.Cfg.FileSettings.ThumbnailWidth)
-	thumbHeight := float64(utils.Cfg.FileSettings.ThumbnailHeight)
-	imgWidth := float64(width)
-	imgHeight := float64(height)
+	thumbWidth := 120
+	thumbHeight := 100
```

Comment:
Can you change these to be constants at the top of the file? Something like `IMAGE_THUMBNAIL_PIXEL_WIDTH` Makes it a little clearer.

Also please make the same changes for the other instances?